lib/src/text-input/TextInput.tsx

```diff
@@ -248,12 +285,15 @@ const DxcTextInput = React.forwardRef<RefType, TextInputPropsType>(
           break;
       }
     };
-    const handleWheel = useCallback((event: WheelEvent) => {
-      if (document.activeElement === inputRef.current) {
-        event.preventDefault();
-        event.deltaY < 0 ? incrementNumber(inputRef.current.value) : decrementNumber(inputRef.current.value);
-      }
-    }, []);
+    const handleWheel = useCallback(
```

Comment:
Even though you are wrapping `handleWheel` in a `useCallback` hook, it will re-render anyway because increment and decrement functions are reassigned in each re-render.

Reply:
You are right.